### PR TITLE
Updated node version in pom.xml

### PR DESCRIPTION
**Pull request by apetkau**
## Description of changes
Updates the version of `nodejs` found in the `pom.xml` file so that `yarn` works properly. The previous version of node was leading to errors like:

```
[INFO] Error: Required package missing from disk. If you keep your packages inside your repository then restarting the Node process may be enough. Otherwise, try to
 run an install first.
```

## Related issue
None.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* ~[ ] Tests added (or description of how to test) for any new features.~
* ~[ ] User documentation updated for UI or technical changes.~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Changes
 * [UI]: Sorted columns for advanced charts and added labels to tiny charts on admin statistics page.
 * [UI]: Fixed: Allow organism name not included in the current taxonomy.
 * [Developer]: Removed `prop-types` as a front-end development dependency.
+* [Developer]: Updated nodejs to `v16.14.2` in the `pom.xml` file.
```
Comment (joshsadam) [inline CHANGELOG.md:21]: why to this version, the current LTS version is 16.15.0?  Did that just get updated?

Comment (apetkau) [reply CHANGELOG.md:21]: Looks like `16.15.0` just got released yesterday https://github.com/nodejs/node/releases. I can update to that one.

Comment (joshsadam) [reply CHANGELOG.md:21]: Please.

**Comment by apetkau** [comment]
I kept on having issues with my `yarn.lock` file getting changed whenever I ran `yarn install` or `mvn clean package`.

Turns out there was a file `~/.yarnrc` in my home directory which was overriding the version of yarn specified in the IRIDA `src/main/webapp` directory. Removing this file fixed the issue.

**Review by joshsadam — APPROVED** [review]
Thanks @apetkau and @ericenns

**Pull request merged**